lxd/cgroup/cgroup.go

```diff
@@ -0,0 +1,69 @@
+package cgroup
+
+import (
+	"fmt"
+	"gopkg.in/lxc/go-lxc.v2"
```

Comment:
I think it's customary that we leave a \n between native go imports and our imports.